aviary/variable_info/variable_meta_data.py

```diff
@@ -3374,6 +3434,30 @@
 #           |___/
 # ====================================================================
 
+add_meta_data(
+    Aircraft.Hydraulics.FC_MASS_COEFFICIENT,
```

Comment:
```suggestion
    Aircraft.Hydraulics.FLIGHT_CONTROL_MASS_COEFFICIENT,
```

Reply:
If this variable gets changed, the new name needs to get updated throughout the code